### PR TITLE
volume_stats: fix lint

### PR DESCRIPTION
**Pull request by keydrevolta**
**What this PR does / why we need it**:
Fix openebs/openebs#770

### CODE REVIEW

**Comment by prateekpandey14** [comment]
Thanks @keydrevolta for fixing linting issues.

**Comment by keydrevolta** [comment]
@prateekpandey14 you're welcome! 🎉

File: command/volume_stats.go [hunk 1]
```diff
@@ -324,29 +324,29 @@ func NewControllerClient(address string) (*ControllerClient, error) {
 	}, nil
 }
 
-// GetStatus will return json response and statusCode
-func GetVolumeStats(address string, obj interface{}) (error, int) {
+// GetStatuStats will return json response and statusCode
```
Comment (utkarshmani1997) [inline command/volume_stats.go]: @keydrevolta can you fix the comments please

Comment (keydrevolta) [reply command/volume_stats.go]: Done :D

**Pull request merged**